shortfin/tests/apps/llm/components/messages_test.py

```diff
@@ -21,17 +21,19 @@ def test_inference_exec_request_repr(mock_void_future):
 
     Patches shortfin.VoidFuture with a mock because we're not running this testcase on a worker thread.
     """
-    req = LlmInferenceExecRequest(InferencePhase.PREFILL, [1, 2, 3, 4], rid="test123")
+    req = InferenceExecRequest(InferencePhase.PREFILL, [1, 2, 3, 4], rid="test123")
```

Comment:
I think kyle's refactoring renamed InferenceExecRequest to LlmInferenceExecRequest